### DISCUSSION TITLE
Sui Framework分析 - Chris's blog

### DISCUSSION THREAD

**Discussion by giscus** [chrisyy2003/chrisyy2003.github.io#16] [评论]
# Sui Framework分析 - Chris's blog

https://github.com/MystenLabs/sui/tree/main/crates/sui-framework/sources sui介绍 头等舱研报： 315-SUI.pdf 架构  每个 Objects 在 Sui 执行环境中都有一个唯一的 ID，并有指向所有者地址的内部指针。通过使用这些概念，很容易通过检查交易是否使用相同的 Objects 来识别关联。 通过将声明关联关系

https://blog.chrisyy.top/intro-sui-framework.html

**Comment by garrymie** [comment]
hi

**Reply by garrymie** [reply]
hi

**Reply by chrisyy2003** [reply]
你好

**Comment by garrymie** [comment]
111

**Comment by chrisyyvip** [comment]
111
